frontend/package.json

```diff
@@ -33,19 +33,19 @@
     "ngx-valdemort": "1.0.0",
     "rxjs": "6.5.2",
     "tslib": "1.9.3",
-    "zone.js": "0.9.1"
+    "zone.js": "~0.9.1"
```

Comment:
You have a few `~` and `^` introduced by the update schematics

Reply:
Thanks. Will fix.